src/Altinn.Platform/Altinn.Platform.Events/Events/Models/EventsSubscription.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Events.Models
+{
+    /// <summary>
+    /// Class that describes a events subscriptions
+    /// </summary>
+    public class EventsSubscription
```

Comment:
No biggie here either, but as it's in the events namespace could the model be named just `subscription` ?

Reply:
fixed